index.html

```diff
@@ -47,8 +47,189 @@ <h1 class="h1">I'm Samson<br>Glad to see<br>you!</h1>
+                        </p>
+                    </div>
+                    <div class="small-btn-div">
+                        <div class="small-btn"><a>html</a></div>
+                        <div class="small-btn"><a>css</a></div>
+                        <div class="small-btn"><a>javaScript</a></div>
+                    </div>
+                    <button><a>See Project</a></button>
+                </div>
+            </div>
+            <div class="grid-item grid-item-4">
+                <div class="card-box">
+                    <img src="images/Snapshoot Portfolio4.png" alt="" srcset="">
+                </div>
+                <div class="text-box">
+                    <h2>Multi-Post<br>Stories</h2>
+                    <div class="canopy">
+                        <h4>CANOPY</h4>
+                        <ul class="back">
+                            <li><img class="dot" src="images/Counter.png" alt="dot">Back End Dev</li>
+                            <li><img class="dot" src="images/Counter.png" alt="dot">2015</li>
+                        </ul>
+                    </div>
+                    <div>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                    </div>
+                    <div class="small-btn-div">
+                        <div class="small-btn"><a>html</a></div>
+                        <div class="small-btn"><a>css</a></div>
+                        <div class="small-btn"><a>javaScript</a></div>
+                    </div>
+                    <button><a>See Project</a></button>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="about">
+        <div class="grid-container">
+            <div class="footer-text">
+                <h2>About<br>Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <div class="con">
+                    <a class="connect">
+                      LET'S CONNECT
+                    </a>
+                </div>
+                <ul class="social">
+                    <li><a><img src="images/twitter.svg" alt="twitter" class="icon"></a></li>
+                    <li><a><img src="images/Linkedin.svg" alt="Linkedin" class="icon"></a></li>
+                    <li><a><img src="images/Vector-3.svg" alt="Vector-3" class="icon"></a></li>
+                    <li><a><img src="images/Vector-2.svg" alt="Vector-2" class="icon"></a></li>
+                    <li><a><img src="images/Vector-4.svg" alt="Vector-4" class="icon"></a></li>
+                </ul>
+                <button><a>Get my resume</a></button>
```

Comment:
:red_circle: I think it is a bad practice to nest a tag inside a button tag. Buttons are supposed to perform a specific action like submitting a form. When you nest one inside the other, it makes it confusing as to what action you want performed. Kindly remove keep the text and remove the a tag inside the button